akka-docs/src/main/paradox/stream/stream-integrations.md

```diff
@@ -55,11 +55,38 @@ order of the emitted downstream elements (the replies).
 
 ### Sink.actorRefWithAck
 
-The sink sends the elements of the stream to the given `ActorRef` that sends back back-pressure signal.
-First element is always *onInitMessage*, then stream is waiting for the given acknowledgement message
-from the given actor which means that it is ready to process elements. It also requires the given acknowledgement
+The `MessageProcessorActor` processes messages and keeps the state of `tototalMessageProcessed`.
```

Comment:
I think it would be good to start with a general description of `Sink.actorRefWithAck`, and only after that describe the example, starting a new paragraph with clear wording like "For example, let's look at an example where we have a `MessageProcessorActor` that processes messages and keeps the state of `totalMessageProcessed`".

Reply:
I agree